manager/config/multi-controller/manager_patch.yaml

```diff
@@ -0,0 +1,30 @@
+# Copyright 2020 IBM Corp.
```

Comment:
Can be move multi cluster configurations to a subfolder of `config`?

- `manager/config/multicluster/coordinator` for this file
- `manager/config/multicluster/remote` for what you called follower